airflow/providers/amazon/aws/example_dags/example_sqs.py

```diff
@@ -0,0 +1,83 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from airflow import DAG
+from airflow.operators.python import PythonOperator
+from airflow.providers.amazon.aws.hooks.sqs import SQSHook
+from airflow.providers.amazon.aws.operators.sqs import SQSPublishOperator
+from airflow.providers.amazon.aws.sensors.sqs import SQSSensor
+from airflow.utils.dates import days_ago
+
+QUEUE_NAME = 'Airflow-Example-Queue'
+AWS_CONN_ID = 'aws_default'
+
+
+def create_queue_fn(**kwargs):
```

Comment:
Do you consider using [Task Flow APII](https://airflow.apache.org/docs/apache-airflow/stable/tutorial_taskflow_api.html)?

Reply:
I did...and I was torn because the other examples don't use it, but this one would be a good candidate.  Would you recommend I switch it to Task Flow?

Reply:
The Task Flow API is relatively new, a lot of the examples were written even before its introduction, and it isn't worth the effort to convert them all. I'd definitely recommend using it for new examples if it fits the need.

Reply:
OK sounds good.  Let me go ahead and switch to Task Flow and update the PR

Reply:
Update for example DAGs is WIP. https://github.com/apache/airflow/pull/16837
Cc: @josh-fell

Reply:
@john-jac There weren't a lot of provider example DAGs that warranted an update to use the TaskFlow API but #18278 converted the applicable ones.

Reply:
There is also an ongoing effort to transition away from using `days_ago(n)` for `start_date` to a static value as well cleaning up `default_args` usage so they are pertinent for the example. #18597 is an example of these updates for reference.